final_assignment/marius_ilau/Main.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import os
+import urllib
+import argparse
+
+
+def before_install():
+    """ Run the download script """
+    os.system("bash ./download.sh")
+
+
+def install():
+    """ Run the install script """
+    attempts = 0
+    retry_interval = 3
+    command = "bash /home/alex/script.sh"
+    cwd = "/home/alex/"
+    env_variables = {"tuxy": "Tuxy Pinguinescu"}
+    while attempts != 3:
+        os.system("bash /home/alex/script.sh > /home/alex/")
```

Comment:
Toți parametrii aceea îi puteai folosi deja în `subprocess.Popen`. Un exemplu poți găsi în `.tavis/build.py`.